src/modules/core/components/FriendlyName/FriendlyName.tsx

```diff
@@ -57,17 +61,19 @@ const FriendlyUserName = ({
   }, [addressRef, autoShrinkAddress]);
   return (
     <div className={styles.main}>
-      {userDisplayName || (username && `@${username}`) || (
-        <MaskedAddress
-          address={walletAddress}
-          full={!maskedAddress}
-          ref={addressRef}
-        />
-      )}
+      {userDisplayName ||
+        (username && `@${username}`) ||
+        colonyData?.colony.colonyName || (
+          <MaskedAddress
+            address={walletAddress}
+            full={!maskedAddress}
+            ref={addressRef}
+          />
+        )}
```

Comment:
Please make sure to also account for the colony displayName (the same way we account for the userDisplayName)

Here's an example where we have a defined colony name:

![Screenshot from 2021-01-12 11-48-47](https://user-images.githubusercontent.com/1193222/104298291-75b83480-54cc-11eb-9882-c17828234452.png)

Yet the `FriendlyName` component just shows the ens name:

![Screenshot from 2021-01-12 11-48-39](https://user-images.githubusercontent.com/1193222/104298287-74870780-54cc-11eb-8c5d-99d04b9add60.png)